### DISCUSSION TITLE
How to submit Task 1 assignment

### DISCUSSION THREAD

**Discussion by erSuyognp** [SimDynamX/SpaceTeamsPRO#48] [2024 Lunar SAR Competition Q&A] [answered]
We have made a working MM can we submit early to get any feedback? Also How do we actually submit the file. It's in our private repo.

**Comment by connorjak** [comment, marked as answer]
Submission instructions are coming out soon. We may or may not have time to give early feedback before the due date for task 1.

**Reply by SkylerHawkins** [reply]
Have the instructions been posted anywhere for how to submit?

**Reply by erSuyognp** [reply]
yes.
https://docs.google.com/forms/d/e/1FAIpQLSdF4w_QgHxagWldRBh9QM-yQMeNoYQZ8MjdKutB4p2hizqF9w/viewform
